### PR TITLE
Movie Data - retrieving and inserting data from the API

### PR DESCRIPTION
**Pull request by ulyanalakevich**

### CODE REVIEW

File: src/core/App/MovieList.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+const MovieList = (props) => {
+    return (
+        <>
+        {props.movieData.map((movie, index) => (
```
Comment (ursmal1948) [inline src/core/App/MovieList.js]: Instead of passing props we can can use here descructuring and pass what we need {movieData} ;)

**Review by ursmal1948 — DISMISSED** [review]
Later we will try to create seperate slice and saga for getting Popular Movies. Overall great job as for the first week!

**Review by ursmal1948 — DISMISSED** [review]
Good job!

**Review by krzysztof-gronski — APPROVED** [review]
Great job! After deep analysis, I approve all changes ;)

**Review by ursmal1948 — APPROVED** [review]

**Pull request merged**